libs/common/src/lib/policies/project-pitch.policy.ts

```diff
@@ -0,0 +1,11 @@
+import { ProjectPitch } from "@terramatch-microservices/database/entities";
+import { UserPermissionsPolicy } from "./user-permissions.policy";
+
+export class ProjectPitchPolicy extends UserPermissionsPolicy {
+  async addRules() {
+    if (this.permissions.some(permission => permission.startsWith("framework-"))) {
+      this.builder.can("read", ProjectPitch);
+      return;
+    }
```

Comment:
This seems to be missing permissions for any PD levels. I haven't double-checked in the FE PR yet; have we removed any need for PDs to directly fetch pitches? If so, this is OK, but if not we need to address that.